kfs-core/src/main/resources/edu/arizona/kfs/gl/spring-gl.xml

```diff
@@ -22,25 +22,9 @@
             </list>
         </property>
         <property name="jobNames">
-			<list>

```

Comment:
Question: were these properties removed on purpose? ... just making sure we're not missing something